db/migrate/20210910082828_create_answers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAnswers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :answers do |t|
+      t.string :content
+      t.integer :votes
+      t.references :user, null: false, foreign_key: true
```

Comment:
foreign_key: true is it needed with t.references?  please check without it

Reply:
strange, its not required but rails still generated this